local-cluster/tests/local_cluster.rs

```diff
@@ -1638,7 +1644,7 @@ fn test_optimistic_confirmation_violation_detection() {
         sleep(Duration::from_millis(100));
     }
 
-    let exited_validator_info = cluster.exit_node(&entry_point_id);
+    let exited_validator_info = cluster.exit_node(&node_to_restart);
```

Comment:
Should all references to `entry_point_id` be replaced with `node_to_restart`? Like the call to `get_validator_client(&entry_point_id)` above

Reply:
I think that is okay to leave the rest as is because it is used as the cluster client to observe the whole cluster state. I changed everything to node_to_restart in my local space, it works as well.

Reply:
On second thought, I changed everything to node_to_restart to keep it closer to the original test case